tests/phpunit/tests/test-wpml-elementor-media-translation-factory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Class Test_WPML_Elementor_Media_Translation_Factory
+ *
+ * @runTestsInSeparateProcesses
```

Comment:
This might not be needed @adrianosferreira since the classes we overload are from another project.

Reply:
Trying now...